### PR TITLE
chore: 🤖 Share query migrated from dropdown to separate button

### PR DESCRIPTION
**Pull request by mczapkowicz**
[FEE-481](https://metakeen.atlassian.net/jira/software/projects/FEE/boards/10?selectedIssue=FEE-481)

### CODE REVIEW

File: src/components/BrowserQueryMenu/BrowserQueryMenu.tsx [hunk 1]
```diff
@@ -41,6 +45,8 @@ const BrowserQueryMenu: FC<Props> = ({ onEditQuery, onRemoveQuery }) => {
   const [actionsMenu, setActionsMenuVisibility] = useState(false);
   const [actionsTooltip, showActionsTooltip] = useState(false);
   const [settingsTooltip, showSettingsTooltip] = useState(false);
+  const [shareTooltip, showShareTooltip] = useState(false);
```
Comment (apryka) [inline src/components/BrowserQueryMenu/BrowserQueryMenu.tsx:48]: what do you think about refactoring these `useStates` into one?

Comment (apryka) [reply src/components/BrowserQueryMenu/BrowserQueryMenu.tsx:48]: e.g `const [visibile, setVisible] = useState<'menu' | 'actions' | 'settings' | 'share'>(false)`

Comment (mczapkowicz) [reply src/components/BrowserQueryMenu/BrowserQueryMenu.tsx]: Thanks, good idea to refactor this. I changed menu tooltips to use MousePositionedTooltips component, so now it looks  much simpler.

File: src/components/EditorNavigation/EditorNavigation.styles.ts [hunk 2]
```diff
@@ -54,19 +53,11 @@ export const Menu = styled.div`
   display: flex;
   margin-left: auto;
 
-  ${MenuItem} + ${MenuItem} {
+  div + div {
```
Comment (apryka) [inline src/components/EditorNavigation/EditorNavigation.styles.ts:56]: might cause problems as we have almost every component rendered as div, so IMO we should be more specific

```
export const Menu = styled.div`
  display: flex;
  margin-left: auto;

  > div + div {
    margin-left: 10px;
  }
`;
```

or

```
export const Menu = styled.div`
  display: flex;
  margin-left: auto;
  column-gap: 10px;
`;
```

Comment (mczapkowicz) [reply src/components/EditorNavigation/EditorNavigation.styles.ts]: Good idea, I have changed it.

**Review by apryka — APPROVED** [review]

**Pull request merged**